tool2schema/parameter_schema.py

```diff
@@ -36,60 +40,96 @@ def matches(parameter: Parameter) -> bool:
         """
         raise NotImplementedError()
 
-    def _add_type(self, schema: dict):
+    def _get_type(self) -> Union[str, Parameter.empty]:
```

Comment:
What is the benefit of using `Parameter.empty` vs just `Optiona[str]`?

Reply:
I used it to differentiate between "I return None" and "I return nothing, discard it" in `_get_default`, since the default value might be none and still be included in the schema. In the other methods it is just for uniformity, it simplifies the iteration in `to_json`

Reply:
Maybe add a comment about this somewhere?

Reply:
It says something in the docstring but I added a comment with 6d659b4 to make it clearer